### PR TITLE
Backend booking

### PR DESCRIPTION
**Pull request by ihakalanka**

### CODE REVIEW

File: back-end/controllers/locationController.js
Comment (Thilina1999) [inline back-end/controllers/locationController.js:1]: delete in location controller?

File: back-end/controllers/transportationModeController.js
Comment (Thilina1999) [inline back-end/controllers/transportationModeController.js:1]: delete in transportModeContoller ?

File: back-end/middlewares/authMiddleware.js
Comment (Thilina1999) [inline back-end/middlewares/authMiddleware.js:1]: Is this part relevant for Booking? remove changes irrelevant for booking

File: back-end/controllers/locationController.js
Comment (ihakalanka) [reply back-end/controllers/locationController.js:1]: yes

File: back-end/controllers/transportationModeController.js
Comment (ihakalanka) [reply back-end/controllers/transportationModeController.js:1]: yes

File: back-end/middlewares/authMiddleware.js
Comment (ihakalanka) [reply back-end/middlewares/authMiddleware.js:1]: just made changes according to best practices

File: back-end/models/location.js
Comment (Thilina1999) [inline back-end/models/location.js:1]: is this file delete?

File: back-end/utils/tokenValidation.js
Comment (Thilina1999) [inline back-end/utils/tokenValidation.js:1]: Is this part relevant for Booking? remove changes irrelevant for booking

File: back-end/utils/tokenGenerate.js
Comment (Thilina1999) [inline back-end/utils/tokenGenerate.js:1]: Is this part relevant for Booking? remove changes irrelevant for booking

File: back-end/models/location.js
Comment (ihakalanka) [reply back-end/models/location.js:1]: yes

**Review by Thilina1999 — CHANGES_REQUESTED** [review]
19 file changes in just one booking section. And some files are missing check again resolve issues

**Review by Thilina1999 — APPROVED** [review]

**Pull request merged**